### DISCUSSION TITLE
Is it possible to add an overlay over the camera

### DISCUSSION THREAD

**Discussion by PercheEtienne** [react-native-image-picker/react-native-image-picker#2051] [Q&A]
Hello everyone, 

I would like to know if it is possible to add an overlay over the camera? 

Here is an example of what I am looking for!
![Screen Shot 2022-11-15 at 12 17 36 PM](https://user-images.githubusercontent.com/47390745/201906805-ebf0c4bf-0c2d-4fa8-9320-b3af339ce393.png)

thanks

**Comment by GaylordP** [comment]
Hello :)
I'm very interested in this question!
Did you find a way to add an overlay/button on top of the picture taking?

Thanks very much
